packages/aws-cdk-lib/aws-s3/lib/bucket.ts

```diff
@@ -1400,6 +1400,145 @@ export abstract class TargetObjectKeyFormat {
   public abstract _render(): CfnBucket.LoggingConfigurationProperty['targetObjectKeyFormat'];
 }
 
+/**
+ * Specifies which Amazon S3 objects to replicate and where to store the replicas.
+ */
+export interface ReplicationRule {
+  /**
+   * A container for information about the replication destination.
+   */
+  readonly destination: ReplicationDestination;
+
+  /**
+   * Specifying S3 Replication Time Control (S3 RTC),
+   * including whether S3 RTC is enabled and the time when all objects and operations on objects must be replicated.
+   *
+   * @default - S3 Replication Time Control is not enabled
+   */
+  readonly replicationTimeControl?: boolean;
```

Comment:
from the [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-replicationtime.html) it seems we should have the option of specifying time value as well so should we have an interface here that takes in timevalue ( also keeping the scope for additional values other than minute in case its introduced in future) ? 
Reading further, i see the only allowed value is 15, it makes sense to extend if we can have more value going forward in the future for that field(will check with the team and get back on this)

Reply:
Understood. 
In that case, would it be acceptable to define an enum like the following and pass it as an argument of `replicationTimeControl`?

```ts
export TimeControl {
  FIFTEEN_MINUTES: 15
}
```

Reply:
Thanks @badmintoncryer , we can go with enum like class.

Reply:
I've implemented it!